docs/UserGuide.md

```diff
@@ -149,21 +149,48 @@ Flag a client in the HustleBook to mark them as important.
 
 Format: `flag NAME`
 
-* `NAME` input should be a name found in the HustleBook.
-* `NAME` is not case-sensitive.
-* In the event where more than one name is matches `NAME` input, you would need to specify using `INDEX` 
-of the list shown.
+* Flags the client with the specified `NAME`.
+  * `Name` is case-insensitive. e.g. `John` will match `john`.
+  * Only full words will be matched e.g. `Han` will not match `Hans`.
+* If `NAME` matches multiple clients, you will see a list of clients with matching names.
+  * Input the position on the list of the client you wish to edit.
+
+Example:
+
+**Scenario 1**: You have a client named `John Doe`
+
+`flag John Doe` flags client `John Doe` from on HustleBook.
```

Comment:
Nice examples, however, I don't quite understand what this sentence `from on HustleBook` means. It would be better to rephrase this.